controllers/scopeinstance_controller.go

```diff
@@ -346,7 +345,7 @@ func (r *ScopeInstanceReconciler) mapToScopeInstance(obj client.Object) (request
 	scopeInstanceList := &operatorsv1.ScopeInstanceList{}
 
 	if err := r.Client.List(ctx, scopeInstanceList); err != nil {
-		r.logger.Error(err, "error listing scope instances")
+		log.Log.Error(err, "error listing scope instances")
```

Comment:
Nit - since the resource name of a `ScopeInstance` is `scopeinstance` it may be nice to be consistent with how Kubernetes handles resource names and log this `scopeinstances` instead of `scope instances`:
```suggestion
		log.Log.Error(err, "error listing scopeinstances")
```